src/components/DayPicker.jsx

```diff
@@ -179,21 +205,107 @@ export default class DayPicker extends React.Component {
         this.adjustDayPickerHeight();
       }
     }
+
+    if (isFocused !== this.props.isFocused) {
+      if (isFocused) {
+        const focusedDate = this.getFocusedDay(currentMonth);
+        this.setState({
+          showKeyboardShortcuts,
+          focusedDate,
+        });
+      } else {
+        this.setState({ focusedDate: null });
```

Comment:
having a little bit of a hard time tracking focus state and shortcut panel status but I'm curious why we wouldn't update the `showKeyboardShortcuts` state if it changed. Can't easily tell if it's just always controlled in one place so you don't have to worry about it, but from an API perspective on this component itself, it's a little confusing.